### PR TITLE
ARCHITECTURE: remove Packer & Terraform

### PR DESCRIPTION
**Pull request by MarkRatjens**
Spaces is still responsible for preparing payloads for these tools.

Spaces is not responsible for executing commands against these tools.

### CODE REVIEW

File: lib/spaces/engines/packing/space.rb [hunk 1]
```diff
@@ -21,88 +19,21 @@ def by(identifier, klass = default_model_class)
...
-      nil
-    end
-
     def save(model)
       raise PackWithoutImagesError, "Model doesn't have images: #{model.identifier}" unless model.has?(:builders)
 
       ensure_connections_exist_for(model)
       super.tap do
-        path_for(model).join('commit.json').write(model.to_h.to_json)
+        path_for(model).join('payload.json').write(model.to_h.to_json)
```
Comment (filterfish) [inline lib/spaces/engines/packing/space.rb:27]: Can you change this to `template.json` please as that's what packer calls it.

File: lib/spaces/spaces/models/space.rb [hunk 2]
```diff
@@ -61,6 +61,10 @@ def writing_name_for(model)
       "#{path_for(model)}/#{model.file_name}"
     end
 
+    def payload_path_for(model)
+      path_for(model).join('payload.*')
```
Comment (filterfish) [inline lib/spaces/spaces/models/space.rb:65]: I presume this also needs to change to `template.json` in order match the other commit.

File: lib/spaces/engines/packing/space.rb (hunk 1 above)
Comment (MarkRatjens) [reply lib/spaces/engines/packing/space.rb:27]: yep

File: lib/spaces/spaces/models/space.rb (hunk 2 above)
Comment (MarkRatjens) [reply lib/spaces/spaces/models/space.rb:65]: ok

File: lib/spaces/engines/packing/space.rb (hunk 1 above)
Comment (MarkRatjens) [reply lib/spaces/engines/packing/space.rb]: change committed

File: lib/spaces/spaces/models/space.rb (hunk 2 above)
Comment (MarkRatjens) [reply lib/spaces/spaces/models/space.rb]: change committed

**Pull request merged**